cmd/genji/main.go

```diff
@@ -8,6 +8,9 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+var CLIVersion = "development"
+var GenjiVersion = "development"
```

Comment:
I was thinking of storing this in Genji directly, and have the CLI use `genji.Version`. Would that work?

Reply:
I tried doing this but since we are actually building the cli, I am not able to pass the build flags to be set in genji. I will give this another look.

Reply:
Perhaps we can use [runtime/debug.ReadBuildInfo](https://golang.org/pkg/runtime/debug#ReadBuildInfo) to extract the main module version so that builds are easily reproducible?

Reply:
One thing to note about publishing build artifacts with Go modules is that instead of cloning the source we should `go get` tagged release (which should also add the new release to default $GOPROXY).

Reply:
@asdine I have moved `GenjiVersion` into `genji` package. If you feel like what @tie has suggested is a better approach, let me know. I will make the neccessary changes. :smile:

Reply:
@cvhariharan @tie That's a great idea!